data/json/recipes/recipe_others.json

```diff
@@ -967,7 +967,8 @@
       { "id": "SCREW", "level": 1 },
       { "id": "WRENCH", "level": 1 }
     ],
-    "tools": [ [ [ "welder", 200 ], [ "welder_crude", 300 ], [ "toolset", 300 ], [ "soldering_iron", 300 ], [ "oxy_torch", 40 ] ] ],
+    "using": [ [ "soldering_standard", 10 ] ],
```

Comment:
Why is this added here, but not in other recipes?

Reply:
This recipe has electronic parts in it, other similar recipes that were changed don't.